### ISSUE TITLE
Implement API for fetching type name label sprites

### ISSUE THREAD

**Issue by gabeklavans** [PokeAPI/pokeapi#1066]
Since [the sprites themselves have been merged in](https://github.com/PokeAPI/sprites/pull/139) I figured I'd move the issue over here cuz it made more sense to me.

See https://github.com/PokeAPI/sprites/issues/137 for context.

**Comment by Naramsim** [comment]
@gabeklavans would you be willing to work on this?

**Comment by gabeklavans** [comment]
> @gabeklavans would you be willing to work on this? 

Sure! I can look into it within a week or so.

**Comment by gabeklavans** [comment]
Been digging into this. I think I figured out how to add the new sprites to the build.py file for types following this structure:
```json
"sprites": {
    "<generation>": {
          "<game>": {
               "name_icon": "https://..."
            }
      }
}
```

However, now I'm trying to actually define the models (I named it `TypeSprites`) so I can build the DB with these new sprites. When I try to `make build-db` I get `sqlite3.OperationalError: no such table: pokemon_v2_typesprites`; I'm guessing I have to add something to serializers.py? Additionally, will I have to write a migration since I'm adding to the schema for types? Never used django for db stuff, so I'm figuring out how it works as I go
